lib/quby/questionnaires/entities/questionnaire.rb

```diff
@@ -233,8 +233,8 @@ def filter_textvars(given_textvars)
 
         def default_textvars
           textvars.select { |key, textvar| textvar.default.present? }
-                  .map    { |key, textvar| [key, textvar.default] }
-                  .to_h
+            .map    { |key, textvar| [key, textvar.default] }
```

Comment:
punten uitlijnen is in dit geval leesbaarder. 
Plus uitlijning aanpassen van code waar je niet aanzit is maakt een pull-request onnodig lang.